scripts/release.sh

```diff
@@ -2,6 +2,13 @@
 rm -rf starcoin-artifacts/*
 mkdir -p starcoin-artifacts/
 cp -v target/release/starcoin starcoin-artifacts/
+cp -v target/release/starcoin_miner starcoin-artifacts/
+cp -v target/release/starcoin_generator starcoin-artifacts/
+cp -v target/release/mpm starcoin-artifacts/
+cp -v target/release/starcoin_db_exporter starcoin-artifacts/
+cp -v scripts/import_block.sh starcoin-artifacts/
+cp -v scripts/import_snapshot.sh starcoin-artifacts/
```

Comment:
import_snapshot.sh is unuseful. Because we changed the storage.

Reply:
Ok, I'd remove it